### PR TITLE
Logging parameter (updated)

### PR DESCRIPTION
**Pull request by alongreyber**
Added a parameter for log level for the /query and /query_result endpoints. The default level is error.

### CODE REVIEW

**Comment by patrickkwang** [comment]
> Added a parameter for log level for the /query and /query_result endpoints. The default level is error. Also created better error handling for KP requests.

Are we actually changing error handling for KP requests here?

File: tests/helpers/utils.py [hunk 1]
```diff
@@ -1,9 +1,25 @@
 import itertools
 import time
+import json
```
Comment (patrickkwang) [inline tests/helpers/utils.py:3]: How are the changes in this file relevant to the logging parameter?

File: tests/query_planner/ex2_qg.json [hunk 2]
```diff
@@ -1,17 +1,18 @@
 {
   "nodes": {
-    "n0": {"id": "MONDO:0005737"},
-    "n1": {"category": "biolink:BiologicalProcessOrActivity"},
-    "n2": {"category": "biolink:AnatomicalEntity"},
-    "n3": {"category": "biolink:PhenotypicFeature"},
-    "n4": {"category": "biolink:PhenotypicFeature"},
-    "n5": {"id": "MONDO:6801"}
+    "n0": {
```
Comment (patrickkwang) [inline tests/query_planner/ex2_qg.json:3]: How are the changes in this file relevant to the logging parameter?

**Comment by alongreyber** [comment]
No we are not. That's in a separate PR that I am opening soon.

File: tests/query_planner/test_query_planner.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
 from tests.helpers.context import \
     with_registry_overlay, with_norm_overlay
-from tests.helpers.utils import generate_kps, time_and_display
+from tests.helpers.utils import load_kps, generate_kps, time_and_display
```
Comment (patrickkwang) [inline tests/query_planner/test_query_planner.py:14]: How are the changes in this file relevant to the logging parameter?

File: tests/server/test_server.py [hunk 4]
```diff
@@ -5,12 +5,16 @@
 import json
 import os
 import httpx
+from fastapi.responses import Response
```
Comment (patrickkwang) [inline tests/server/test_server.py:8]: Are all of the changes in this file relevant to the logging parameter?

**Review by patrickkwang — CHANGES_REQUESTED** [review]

**Comment by patrickkwang** [comment]
Feel free to force push to this feature branch if necessary rather than opening a new PR.

**Comment by alongreyber** [comment]
The changes to the ex1_qg file and the load_kps change aren't strictly related to this PR. Do you want me to go through and remove those? If so, where should those go, if anywhere?

**Comment by patrickkwang** [comment]
Yes, those changes should not have happened here (which is different from happening and then being removed - yah?). Independent changes should go in independent commits, and a PR should be a set of commits that accomplishes a specific goal. You could present those test refactoring changes in another PR.

File: tests/server/test_server.py [hunk 5]
```diff
@@ -5,12 +5,15 @@
...
 
 from reasoner_pydantic import Query, Message, QueryGraph
 import pytest
 
 from tests.helpers.logger import setup_logger
-from tests.helpers.context import with_translator_overlay
+from tests.helpers.context import \
+    with_translator_overlay, with_registry_overlay, \
```
Comment (patrickkwang) [inline tests/server/test_server.py:15]: I suspect these currently unused imports are relevant to a separate change.

**Review by patrickkwang — APPROVED** [review]
Thanks for indulging me in my git/GitHub stickler-ism. Let's continue trying to leave behind a clean and readable history.

**Pull request merged**